interfaces/Portalicious/src/app/components/page-layout/components/registration-lookup-menu/registration-lookup-menu.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  input,
+} from '@angular/core';
+
+import {
+  injectQuery,
+  injectQueryClient,
+} from '@tanstack/angular-query-experimental';
+import { MenuItem } from 'primeng/api';
+
+import { TabsMenuComponent } from '~/components/tabs-menu/tabs-menu.component';
+import { ProjectApiService } from '~/domains/project/project.api.service';
+import { RegistrationApiService } from '~/domains/registration/registration.api.service';
+import { registrationLink } from '~/domains/registration/registration.helper';
+import { TranslatableStringService } from '~/services/translatable-string.service';
+
+@Component({
+  selector: 'app-registration-lookup-menu',
+  imports: [TabsMenuComponent],
+  templateUrl: './registration-lookup-menu.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RegistrationLookupMenuComponent {
+  readonly phonenumber = input.required<string>();
+
+  readonly projectApiService = inject(ProjectApiService);
+  readonly registrationApiService = inject(RegistrationApiService);
+  readonly translatableStringService = inject(TranslatableStringService);
+
+  readonly queryClient = injectQueryClient();
```

Comment:
I think this pattern just changed via an update to one of the Tanstack-packages...

See #6372

Reply:
Ha, yes, updated it 🙊